pkg/plugins/ifaces.go

```diff
@@ -44,6 +44,10 @@ type Manager interface {
 	GetPluginDashboards(orgID int64, pluginID string) ([]*PluginDashboardInfoDTO, error)
 	// GetPluginMarkdown gets markdown for a certain plugin/name.
 	GetPluginMarkdown(pluginID string, name string) ([]byte, error)
+	// CacheDashboardUIDBySlug is used to map plugin dashboard slugs into UIDs without making additional database queries.
+	CacheDashboardUIDBySlug(orgID int64, slug string, uid string)
+	// GetDashboardUIDBySlug returns a plugin dashboard UID by dashboard slug.
+	GetDashboardUIDBySlug(orgID int64, slug string) string
```

Comment:
Not. Would make more sense to change this to `GetPluginDashboardBySlug(orgID int64, pluginID string, slug string)` or something to mate it similar to `GetPluginDashboards`above.